src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Misc.cs

```diff
@@ -1291,5 +1291,20 @@ public void TFS_470020()
             CError.Compare(errorCount, 1, "Error Count mismatch!");
             return;
         }
+
+        [Fact]
+        public void Issue_60543()
+        {        
+            string xsd = Path.Combine(TestData._Root, "issue60543.xsd");
+            XmlSchemaSet s = new XmlSchemaSet();
+            s.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+            s.XmlResolver = new XmlUrlResolver();
+            XmlReader r = XmlReader.Create(xsd);
+            s.Add(null, r);
+            s.Compile();
+            CError.Compare(warningCount, 0, "Warning Count mismatch!");
+            CError.Compare(errorCount, 0, "Error Count mismatch!");
+            return;
```

Comment:
Old tests have been written badly and we haven't had time to clean up but new ones should be written as good as we can:
- remove redundant `return`
- use `Assert.Equal` rather than `CError.Compare`
- We don't need `XmlUrlResolver` here (likely `s.XmlResolver = new XmlUrlResolver();` is not needed at all)